package com.capstone.domain.user.exception;

public class UserFoundException extends RuntimeException {
    public UserFoundException(String message) {
        super(message);
    }
}

